gmail-oauth2_none_none.sh

```diff
@@ -50,22 +50,33 @@ get_refresh_token() {
   assert_value "client_secret" "missing: client_secret"
 
   local -r scope="https://mail.google.com/"
-  local -r redirect_uri="urn:ietf:wg:oauth:2.0:oob"
+  local -r redirect_uri="https://localhost"
   local -r response_type="code"
 
   echo_err ""
-  echo_err "For authorization code, visit this url and follow the instructions:"
-  echo_err "  https://accounts.google.com/o/oauth2/auth?client_id=${client_id}&redirect_uri=${redirect_uri}&response_type=${response_type}&scope=${scope}"
+  echo_err "1. Ensure that your OAuth 2.0 Client ID has 'https://localhost' registered as an authorized redirect URL here:"
+  echo_err "  https://console.cloud.google.com/apis/credentials"
```

Comment:
This part was a bit confusing for me, because desktop application doesn't need any authorized redirect url.

Maybe the `README.md` should mention that OAuth of type Desktop App should be used?

Reply:
Do you mind if I tweak the changes a bit?